apps/acls/serializers/base.py

```diff
@@ -70,6 +70,9 @@ def set_action_choices(self):
             return
         if not settings.XPACK_LICENSE_IS_VALID:
             field_action._choices.pop(ActionChoices.review, None)
+        if not (settings.XPACK_LICENSE_EDITION_ULTIMATE and settings.FACE_RECOGNITION_ENABLED):
+            field_action._choices.pop(ActionChoices.face_verify, None)
+            field_action._choices.pop(ActionChoices.face_online, None)
         for choice in self.Meta.action_choices_exclude:
             field_action._choices.pop(choice, None)
 
```

Comment:
This is not an issue of a code difference from the given context.
However, it's recommended to always ensure that you keep your license valid and up-to-date, because licenses often have expiration dates like XPACK LICENSE IS VALID. This could affect user actions.

Additionally, setting FACE_RECOGNITION_ENABLED only on the Ultimate version would be better to optimize user experiences since not all users need face verification functionalities but they require some form of authentication for certain tasks such as live video calls. Also make sure to update the code according to any updates in the settings when needed.